src/functions-and-arrays.js

```diff
@@ -1,20 +1,50 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(numb1,numb2) {
+
+  if (numb1 > numb2){
+return numb1;
+}
+else if(numb1 < numb2){
+  return numb2;
+} else if (numb1 === numb2){
+  return numb1 || numb2;
+}
+}
 
```

Comment:
Hi Alejandro!
I won't go into the failed tests so much now, you can look at the solutions in the Student Portal again to see exactly what was asked for.
Task 1 looks good, but you could take care of your indentation or have vscode format the indentation.